docs/hugo/content/introduction/multitenant-deployment.md

```diff
@@ -0,0 +1,78 @@
+# Deploying ASO v2 in multi-tenant mode
+
+The current release version of the ASO deployment YAML installs the operator in single-tenant mode:
+it's deployed in the `azureserviceoperator-system` namespace with one set of Azure credentials, and manages resources in any namespace in the cluster.
+That single operator deployment handles webhooks fired when Azure resources are changed.
+
+ASO can also be deployed in a multi-tenant configuration that enables using separate credentials to manage resources in different Kubernetes namespaces.
+
+Running the operator in multi-tenant mode requires one deployment to handle webhooks (since webhook configurations are cluster-level resources) and then a deployment for each tenant, each with its own credentials and set of namespaces that it watches for Azure resources.
+To deploy the operator in multi-tenant mode the release YAML has been split into two parts:
+
+1. **Cluster-wide resources**:
+   * Custom resource definitions for the Azure resources.
+
+   * Cluster roles for managing those resources.
+
+   * The `azureserviceoperator-system` namespace containing the deployment and service to handle ASO webhooks.
+     The webhook service is a deployment of the ASO operator image, but configured to run in webhook-only mode.
+     It won't try to reconcile Azure resources with ARM, and so doesn't need Azure credentials.
```

Comment:
Any acronym should be expanded in place and the sentence should make sense.

```suggestion
   * The `azureserviceoperator-system` namespace containing the deployment and service to handle ASO webhooks.
     The webhook service is a deployment of ASO, but configured to run in _webhook-only_ mode.
     It will not try to reconcile Azure resources with ARM, and so does not need any Azure credentials.
```

Reply:
Good call about the acronym, I've added that early in the doc. I'm keeping the distinction that the webhook service is a deployment of the ASO image based on a comment from @matthchr - he felt that calling the webhook service a deployment of the operator was a bit confusing and I agree.

I'm keeping the contractions because I prefer them! 😝